pg_views/management/commands/show_view_to_model_diff.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management.base import BaseCommand
+from hc_pg_views.models import HCVersionView, HCView
+import django.apps
+from django.conf import settings
+
+
+def strip_view(name):
+    return name[:-len('View')]
+
+def strip_version_view(name):
+    return name[:-len('VersionView')]
+
+
+class Command(BaseCommand):
+
+    def handle(self, **options):
+        views = [strip_view(x.__name__) for x in HCView.__subclasses__(
+        )] + [strip_version_view(x.__name__) for x in HCVersionView.__subclasses__()] + settings.HC_DEACTIVATED_VIEWS
```

Comment:
HCVersionView by  mělo dědit od HCView nebo __subclasses__ vracejí pouze přímého potomka?

Reply:
nemůžeš používat HCView když není ani v této knihovně definované... Musíš používat View

Reply:
koukni se do souboru loading je tam metoda get_sql_model_views

Reply:
ne HC_DEACTIVATED_VIEWS ale jen DEACTIVATED_VIEWS

Reply:
subclasses vraceji jenom primeho potomka